core/src/components/select/test/highlight/index.html

```diff
@@ -117,6 +124,30 @@ <h2>Invalid</h2>
           </div>
         </div>
 
+        <h1>No Fill, Custom</h1>
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Focus</h2>
+            <ion-select label-placement="start" value="hi@ionic.io" class="custom ion-focused" label="Fruit">
+              <ion-select-option value="apple">Apple</ion-select-option>
+            </ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Valid, Focus</h2>
+            <ion-select label-placement="start" class="custom ion-valid ion-focused" label="Fruit">
+              <ion-select-option value="apple">Apple</ion-select-option>
+            </ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Invalid</h2>
```

Comment:
For some reason, the height isn't being changed for the invalid select when running locally. The same is happening for the others.

![Screenshot 2024-03-04 at 5 04 00 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/5e85d5f8-7f1d-4bcc-985f-51f2ae50c84d)

Reply:
This is similar to how input and textarea works that I mentioned here: https://github.com/ionic-team/ionic-framework/pull/29090/files#r1518139609

The difference is select shows the highlight when the `select-expanded` or `ion-focused` class is added:

```css
:host(.select-expanded) .select-highlight,
:host(.ion-focused) .select-highlight {
  transform: scale(1);
}
```

The invalid examples don't have either of these classes, but the other examples have `ion-focused`, which is why the highlight doesn't show unless you tab to the select or open it.